src/main/java/util/FormplayerDateUtils.java

```diff
@@ -0,0 +1,28 @@
+package util;
+
+import java.text.DateFormat;
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+import java.util.Locale;
+import java.util.TimeZone;
+
+/**
+ * Date utility functions for dealing with dates in formplayer
+ */
+public class FormplayerDateUtils {
+    public static String convertJavaDateStringToISO(String date) {
+        DateFormat dfFrom = new SimpleDateFormat("EEE MMM dd kk:mm:ss z yyyy", Locale.ENGLISH);
```

Comment:
probably should have caught this earlier, but eventually we should be storing our dates in ISO format. it's going to leave us headaches down the line. currently we just `date.toString()` which java outputs as some non-iso standard date. that means when we send any dates anywhere else it's hard to parse. in this case i convert it to iso string so that the browser can more easily handle it